text/0148-sampling-probability.md

```diff
@@ -0,0 +1,209 @@
+= 1000`.
+
+## Internal details
+
+The statistical foundation of this technique is known as the
+Horvitz-Thompson estimator ("A Generalization of Sampling Without
+Replacement From a Finite Universe", JSTOR 1952).  The
+Horvitz-Thompson technique works with _unequal probability sampling_
+designs, enabling a variety of techniques for controlling properties
+of the sample.  
+
+For example, you can sample 100% of error events while sampling 1% of
+non-error events, and the interpretation of `sample_count` will be
+correct.
+
+### Bias, variance, and sampling errors
+
+There is a fundamental tradeoff between bias and variance in
+statistics.  The use of unbiased sampling leads unavoidably to
+increased variance.
+
+Estimating sampling errors and variance is out of scope for this
+proposal.  We are satisfied with the unbiased property, which guarantees
+that the expected value of totals derived from the sample equals the
+true population totals.  This means that statistics derived from
+sample data in this way are always accurate, and that more data will
+always improve precision.
+
+### Non-probabilistic rate-limiters
+
+Rate-limiting stages in a telemetry collection pipeline interfere with
+sampling schemes when they operate in non-probabilistic ways.  When
+implementing a non-probabilistic form of rate-limiting, processors
+MUST set the `sample_count` to zero.
+
+### No Sampler configured
+
+When no Sampler is in place and all telemetry events pass to the
+output, the `sample_count` field SHOULD be set to 1 to indicate
+perfect representivity, indicating that no sampling was performed.
+
+### Applicability for tracing
+
+The sampling approach covered here dictates how to select root spans
+in a probabilistic way.  When recording root spans, the `sample_count`
+field should be set as described above.  The adjusted `sample_count`
+of the root span applies the trace, meaning the trace should be
+considered as representative of `sample_count` traces in the
+population.
+
+When non-root spans are recorded because they are part of an ongoing
+trace, they are considered non-probabilistic exemplars.  Non-root
+spans should have `sample_count` set to zero.
+
+### Applicability for metrics
+
+The use of sampling in metrics makes it possible to record
+high-cardinality metric events efficiently, as demonstrated by Statsd.
+
+By pushing sampled metric events from client to server, instead of
+timeseries, it is possible to defer decisions about cardinality
+reduction to the server, without unreasonable cost to the client.
+
+## Prior art and alternatives
+
+The name `sample_count` is proposed because the resulting value is
+effectively a count and may be used in place of the exact count.
+
+Statsd conveys inclusion probability instead of `sample_count`, where
+it is often called "sample rate".
+
+Another name for the proposed `sample_count` field is
+`inverse_probability`, which is considered less suggestive of the
+field's purpose.
```

Comment:
I find the latter to be clearer within the topic of sampling, though not fit for purpose here. I would prefer `sampling_factor` or `compression_factor`, because it's more general than "count"—in that it need not be integral—and it suggests multiplying by it to find another value.

Reply:
I've removed this name from my proposal. I'm using the term "adjusted count" to explain the concept, and would like you to review and see how this reads now.